### PR TITLE
actions: improve GitHub actions reply

### PR DESCRIPTION
**Pull request by poyea**
### Description

Use the default GIF in [the org page](https://github.com/dev-protocol) to replace the image **for now**.

### Why is this change needed?

Refer to #103

### Related Issues

#103

### Code of Conduct

- [x] By submitting this pull request, I confirm I've read and complied with the [CoC](https://github.com/dev-protocol/community/blob/main/CODE_OF_CONDUCT.md) 🖖

### CODE REVIEW

File: .github/workflows/invitation.yml [hunk 1]
```diff
@@ -12,6 +12,6 @@ jobs:
...
+          comment: "![image](https://raw.githubusercontent.com/dev-protocol/.github/main/assets/Dev--animated.gif)<br><b>Invitation sent to join the GitHub Organization. Welcome to Dev Protocol 🎉</b><br><br>Please check your mail to accept the invitation sent by the bot! 🤖<br><h1>About us</h1>Dev Protocol is a solution for decentralized creator economy, it is designed for the blockchain solution and enhance creators to create their economy 💻 Great to have you in our community ✨ Looking forward to your valuable contributions! 🤓 You can also check our community [socials](https://bio.link/devprotocol) ✨ 💛 You can also share our community with other folks who want to learn Blockchain, tech, and contribute together and with us 👉👈"
```
Comment (Panquesito7) [inline .github/workflows/invitation.yml:15]: What if we use this?

image

```suggestion
          comment: "![image](https://user-images.githubusercontent.com/51391473/138764839-1011988e-66f2-4427-a8fc-e0380971df29.gif)<br><b>Invitation sent to join the GitHub Organization. Welcome to Dev Protocol 🎉</b><br><br>Please check your mail to accept the invitation sent by the bot! 🤖<br><h1>About us</h1>Dev Protocol is a solution for decentralized creator economy, it is designed for the blockchain solution and enhance creators to create their economy 💻 Great to have you in our community ✨ Looking forward to your valuable contributions! 🤓 You can also check our community [socials](https://bio.link/devprotocol) ✨ 💛 You can also share our community with other folks who want to learn Blockchain, tech, and contribute together and with us 👉👈"
```

**Review by Panquesito7 — COMMENTED** [review]
Great catch 👀 Thanks for making a PR! 😃

File: .github/workflows/invitation.yml (hunk 1 above)
Comment (poyea) [reply .github/workflows/invitation.yml]: Good!

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contributions! 😄👍

**Review by vinzvinci — APPROVED** [review]
LGTM 👍

**Pull request merged**